### PR TITLE
Update README.md (Removed GPT4ALL added Luna AI Llama2 and how to pages)

### PR DESCRIPTION
**Pull request by lunamidori5**
**Description**

This PR fixes #

**Notes for Reviewers**
Per Request on Discord, updating docs.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ x] Yes, I signed my commits.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,15 +104,17 @@ Note that this started just as a [fun weekend project](https://localai.io/#backs
 
 Check out the [Getting started](https://localai.io/basics/getting_started/index.html) section in our documentation.
 
-### 💡 Example: Use GPT4ALL-J model
+### 💡 Example: Use Luna AI Llama2 model
 
-See the [documentation](https://localai.io/basics/getting_started/#example-use-gpt4all-j-model-with-docker-compose)
```
Comment (mudler) [inline README.md:109]: I'd rather suggest to have a quick example without any specific setup needed ( just copy the model in the folder and done ), from there we can link then to the how-to's

Comment (lunamidori5) [reply README.md:109]: Alright how would you like that done?

Comment (lunamidori5) [reply README.md:109]: point to the all in one demo?

**Comment by mudler** [comment]
as discussed in Discord, let's close this one with the feedback from review

**Pull request closed without merging**